boolean-5.js

```diff
@@ -0,0 +1,24 @@
+/*
+After a hard quarter in the office you decide to get some rest on a vacation. So you will book a flight for you and your girlfriend and try to leave all the mess behind you.
+You will need a rental car in order for you to get around in your vacation. The manager of the car rental makes you some good offers.
+Every day you rent the car costs $40. If you rent the car for 7 or more days, you get $50 off your total. Alternatively, if you rent the car for 3 or more days, you get $20 off your total.
+Write a code that gives out the total amount for different days(d).
+ */
+
+const dayPrice = 40
+
+function rentalCarCost(days) {
+    if (days >= 7)
+        return (days * dayPrice) - 50
+    else if (days >= 3 && days < 7)
+        return (days * dayPrice) - 20
+    else if (days < 3)
+        return (days * dayPrice)
```

Comment:
kod dziala ale czy potrzeba else skoro jest return? czemu do if'ów nie dajesz `{}` nie zwieksza to wg ciebie czytelności?

Reply:
Przyznaje racje, niepotrzebne. Wrzucam skróconą wersje